component/org.wso2.carbon.identity.api.otp.service.smsOtp/pom.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Even an XML file should contain WSO2 license header.

Reply:
fixed in 69f620f